cmd/minikube/cmd/start_flags.go

```diff
@@ -191,6 +192,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().Int(extraDisks, 0, "Number of extra disks created and attached to the minikube VM (currently only implemented for hyperkit and kvm2 drivers)")
 	startCmd.Flags().Duration(certExpiration, constants.DefaultCertExpiration, "Duration until minikube certificate expiration, defaults to three years (26280h).")
 	startCmd.Flags().String(binaryMirror, "", "Location to fetch kubectl, kubelet, & kubeadm binaries from.")
+	startCmd.Flags().Bool(disableOptimizations, false, "If set, disables optimizations that improve minikube performance. Defaults to false.")
```

Comment:
how about list all the optimizations that it disables

corne Dns Replica , House Cleaning...

Reply:
Will do

Reply:
Done